src/lib/services/repository-access/RepositorySearchService.ts

```diff
@@ -37,23 +38,31 @@ export class MultipleRepositorySearchService {
         });
         const log = Log.create();
         const prismaConnector = PrismaConnector.create();
-        return new MultipleRepositorySearchService(repositoryClient, submodelRepositoryClient, prismaConnector, log);
+        return new RepositorySearchService(repositoryClient, submodelRepositoryClient, prismaConnector, log);
     }
 
     static createNull({
         shellsSavedInTheRepositories = [],
         submodelsSavedInTheRepository = [],
         log = null,
-    }: NullableMultipleDataSourceSetupParameters = {}): MultipleRepositorySearchService {
+    }: NullableMultipleDataSourceSetupParameters = {}): RepositorySearchService {
         const aasUrls = [...new Set(shellsSavedInTheRepositories?.map((entry) => entry.repositoryUrl))];
-        return new MultipleRepositorySearchService(
+        return new RepositorySearchService(
             AssetAdministrationShellRepositoryApi.createNull({ shellsSavedInTheRepositories }),
             SubmodelRepositoryApi.createNull({ submodelsSavedInTheRepository }),
             PrismaConnector.createNull({ aasUrls }),
             log ?? Log.createNull(),
         );
     }
 
+    async getAasFromDefaultRepository(aasId: string): Promise<AssetAdministrationShell> {
+        try {
```

Comment:
Why are we sometimes try catching and sometimes not? My first impression would be that consistency would be good. What keeps us from having it throw at a lower level all the time?

Reply:
For consistency, I made all functions catch, good catch :D.